core/src/banking_stage.rs

```diff
@@ -2977,7 +2992,11 @@ mod tests {
                 0,
                 None,
                 &gossip_vote_sender,
-                &QosService::new(Arc::new(RwLock::new(CostModel::default())), 1),
+                &QosService::new(
```

Comment:
QosService::default might be a nice way to clean this up

Reply:
I will add a new function default_for_test so that we are conscious that we only use it for tests.